### DISCUSSION TITLE
Not sure what to focus on in my project

### DISCUSSION THREAD

**Discussion by Ruijia00** [Part-II-Computational-Physics/.github#102] [Q&A] [answered]
Hi!

I'm doing the project 'The wavefunction collapse algorithm'. In the process, I found myself a bit sidetracked, spending a lot of time investigating the properties of the images I've generated (e.g. quantifying how patterns change while adjusting parameters). I've only implemented one algorithm so far.
Is this a bad thing? Should I spend more time focusing on implementing different algorithms and comparing their performance?

Thanks!

**Comment by AustenLamacraft** [comment, marked as answer]
It's OK to focus on one algorithm and investigate how it behaves.

One direction which has interesting connections to physics is the question of how the performance of the algorithm varies as a function of some parameter in the tile set see e.g. [this Twitter thread](https://twitter.com/ExUtumno/status/1119996186663030784). 

You could also look at generating [Domino](https://en.wikipedia.org/wiki/Domino_tiling) or Tatami tilings.

**Reply by Ruijia00** [reply]
Thank you!
